cli/src/services/document.ts

```diff
@@ -46,6 +46,7 @@ export default class Document {
     this.projectId = config.project;
     this.target = this.config.target;
     this.paths = new Tree(this.config).list();
+    this.extraHeaders = config.extraHeaders;
```

Comment:
This is already assigned on line 45, what is the difference here beside making it "undefined"?